prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -103,6 +104,8 @@ type Options struct {
 	OncallGroup string `yaml:"onCallGroup"`
 	// Whether to skip creating the pull request for this bump.
 	SkipPullRequest bool `yaml:"skipPullRequest"`
+	// Information needed to do a gerrit bump. Do not include if doing github bump
+	Gerrit Gerrit `yaml:"gerrit"`
```

Comment:
Can we make this a pointer like `Gerrit *Gerrit  `, then later on you don't need the bool `GerritBump`?

Reply:
Done